diff --git a/node_modules/allotment/dist/modern.mjs b/node_modules/allotment/dist/modern.mjs
index d5dbdc6..e2d9d2e 100644
--- a/node_modules/allotment/dist/modern.mjs
+++ b/node_modules/allotment/dist/modern.mjs
@@ -1156,11 +1156,11 @@ class Ne extends W {
     return e < 0 || e >= this.viewItems.length ? -1 : this.viewItems[e].size;
   }
   isViewVisible(e) {
-    if (e < 0 || e >= this.viewItems.length) throw new Error("Index out of bounds");
+    if (e < 0 || e >= this.viewItems.length) return false;
     return this.viewItems[e].visible;
   }
   setViewVisible(e, t) {
-    if (e < 0 || e >= this.viewItems.length) throw new Error("Index out of bounds");
+    if (e < 0 || e >= this.viewItems.length) return;
     this.viewItems[e].setVisible(t), this.distributeEmptySpace(e), this.layoutViews(), this.saveProportions();
   }
   distributeViewSizes() {
diff --git a/node_modules/allotment/dist/module.js b/node_modules/allotment/dist/module.js
index 0de655f..f296aff 100644
--- a/node_modules/allotment/dist/module.js
+++ b/node_modules/allotment/dist/module.js
@@ -1168,11 +1168,11 @@ class Ne extends W {
     return e < 0 || e >= this.viewItems.length ? -1 : this.viewItems[e].size;
   }
   isViewVisible(e) {
-    if (e < 0 || e >= this.viewItems.length) throw new Error("Index out of bounds");
+    if (e < 0 || e >= this.viewItems.length) return false;
     return this.viewItems[e].visible;
   }
   setViewVisible(e, t) {
-    if (e < 0 || e >= this.viewItems.length) throw new Error("Index out of bounds");
+    if (e < 0 || e >= this.viewItems.length) return;
     this.viewItems[e].setVisible(t), this.distributeEmptySpace(e), this.layoutViews(), this.saveProportions();
   }
   distributeViewSizes() {
