diff --git a/node_modules/@ammarahmed/react-native-share-extension/android/build.gradle b/node_modules/@ammarahmed/react-native-share-extension/android/build.gradle
index 130996e..c245cf9 100644
--- a/node_modules/@ammarahmed/react-native-share-extension/android/build.gradle
+++ b/node_modules/@ammarahmed/react-native-share-extension/android/build.gradle
@@ -1,5 +1,5 @@
 buildscript {
-  ext.kotlin_version = '1.4.0'
+  ext.kotlin_version = '1.6.10'
   repositories {
     jcenter()
     google()
