diff --git a/node_modules/@flyerhq/react-native-link-preview/lib/utils.js b/node_modules/@flyerhq/react-native-link-preview/lib/utils.js
index c1aff89..4a3f878 100644
--- a/node_modules/@flyerhq/react-native-link-preview/lib/utils.js
+++ b/node_modules/@flyerhq/react-native-link-preview/lib/utils.js
@@ -1,4 +1,4 @@
-import { decode } from 'html-entities';
+import { decode } from 'entities';
 import { Image } from 'react-native';
 export const getActualImageUrl = (baseUrl, imageUrl) => {
     let actualImageUrl = imageUrl === null || imageUrl === void 0 ? void 0 : imageUrl.trim();
