diff --git a/node_modules/@notifee/react-native/dist/NotifeeApiModule.js b/node_modules/@notifee/react-native/dist/NotifeeApiModule.js
index f9f151a..d31b0d1 100644
--- a/node_modules/@notifee/react-native/dist/NotifeeApiModule.js
+++ b/node_modules/@notifee/react-native/dist/NotifeeApiModule.js
@@ -275,6 +275,7 @@ class NotifeeApiModule extends NotifeeNativeModule_1.default {
         return Promise.resolve();
     };
     createTriggerNotification = (notification, trigger) => {
+
         let options;
         let triggerOptions;
         try {
diff --git a/node_modules/@notifee/react-native/dist/types/Trigger.d.ts b/node_modules/@notifee/react-native/dist/types/Trigger.d.ts
index 9907278..89d2116 100644
--- a/node_modules/@notifee/react-native/dist/types/Trigger.d.ts
+++ b/node_modules/@notifee/react-native/dist/types/Trigger.d.ts
@@ -55,7 +55,8 @@ export declare enum RepeatFrequency {
     NONE = -1,
     HOURLY = 0,
     DAILY = 1,
-    WEEKLY = 2
+    WEEKLY = 2,
+    MONTHLY = 3
 }
 /**
  * Interface for building a trigger that repeats at a specified interval.
diff --git a/node_modules/@notifee/react-native/dist/types/Trigger.js b/node_modules/@notifee/react-native/dist/types/Trigger.js
index 6fe5153..0043d15 100644
--- a/node_modules/@notifee/react-native/dist/types/Trigger.js
+++ b/node_modules/@notifee/react-native/dist/types/Trigger.js
@@ -12,6 +12,7 @@ var RepeatFrequency;
     RepeatFrequency[RepeatFrequency["HOURLY"] = 0] = "HOURLY";
     RepeatFrequency[RepeatFrequency["DAILY"] = 1] = "DAILY";
     RepeatFrequency[RepeatFrequency["WEEKLY"] = 2] = "WEEKLY";
+    RepeatFrequency[RepeatFrequency["MONTHLY"] = 3] = "MONTHLY";
 })(RepeatFrequency = exports.RepeatFrequency || (exports.RepeatFrequency = {}));
 /**
  * An interface representing the different units of time which can be used with `IntervalTrigger.timeUnit`.
diff --git a/node_modules/@notifee/react-native/ios/NotifeeCore/NotifeeCoreUtil.h b/node_modules/@notifee/react-native/ios/NotifeeCore/NotifeeCoreUtil.h
index 9ae89e3..ecc7afa 100644
--- a/node_modules/@notifee/react-native/ios/NotifeeCore/NotifeeCoreUtil.h
+++ b/node_modules/@notifee/react-native/ios/NotifeeCore/NotifeeCoreUtil.h
@@ -41,7 +41,8 @@ typedef NS_ENUM(NSInteger, NotifeeCoreTriggerType) {
 typedef NS_ENUM(NSInteger, NotifeeCoreRepeatFrequency) {
   NotifeeCoreRepeatFrequencyHourly = 0,
   NotifeeCoreRepeatFrequencyDaily = 1,
-  NotifeeCoreRepeatFrequencyWeekly = 2
+  NotifeeCoreRepeatFrequencyWeekly = 2,
+  NotifeeCoreRepeatFrequencyMonthly = 3
 };
 
 @interface NotifeeCoreUtil : NSObject
diff --git a/node_modules/@notifee/react-native/ios/NotifeeCore/NotifeeCoreUtil.m b/node_modules/@notifee/react-native/ios/NotifeeCore/NotifeeCoreUtil.m
index e489445..71b360e 100644
--- a/node_modules/@notifee/react-native/ios/NotifeeCore/NotifeeCoreUtil.m
+++ b/node_modules/@notifee/react-native/ios/NotifeeCore/NotifeeCoreUtil.m
@@ -337,6 +337,10 @@ + (UNNotificationTrigger *)timestampTriggerFromDictionary:(NSDictionary *)trigge
       // match by day, hour, minute, and second
       calendarUnit =
           NSCalendarUnitWeekday | NSCalendarUnitHour | NSCalendarUnitMinute | NSCalendarUnitSecond;
+    } else if (repeatFrequency == NotifeeCoreRepeatFrequencyMonthly) {
+      // match by day, hour, minute, and second
+      calendarUnit =
+          NSCalendarUnitDay | NSCalendarUnitHour | NSCalendarUnitMinute | NSCalendarUnitSecond;
     } else {
       NSLog(@"NotifeeCore: Failed to parse TimestampTrigger with unknown "
             @"repeatFrequency: %ld",
diff --git a/node_modules/@notifee/react-native/src/types/Trigger.ts b/node_modules/@notifee/react-native/src/types/Trigger.ts
index 6c0760a..5ad765b 100644
--- a/node_modules/@notifee/react-native/src/types/Trigger.ts
+++ b/node_modules/@notifee/react-native/src/types/Trigger.ts
@@ -60,6 +60,7 @@ export enum RepeatFrequency {
   HOURLY = 0,
   DAILY = 1,
   WEEKLY = 2,
+  MONTHLY = 3
 }
 
 /**
