diff --git a/node_modules/@react-native-community/cli-plugin-metro/build/tools/loadMetroConfig.d.ts b/node_modules/@react-native-community/cli-plugin-metro/build/tools/loadMetroConfig.d.ts
index 6cbcc22..62d5422 100644
--- a/node_modules/@react-native-community/cli-plugin-metro/build/tools/loadMetroConfig.d.ts
+++ b/node_modules/@react-native-community/cli-plugin-metro/build/tools/loadMetroConfig.d.ts
@@ -12,6 +12,7 @@ export interface ConfigOptionsT {
     reporter?: any;
     config?: string;
 }
+
 /**
  * Load Metro config.
  *
diff --git a/node_modules/@react-native-community/cli-plugin-metro/build/tools/loadMetroConfig.js b/node_modules/@react-native-community/cli-plugin-metro/build/tools/loadMetroConfig.js
index f26ef52..233a7c9 100644
--- a/node_modules/@react-native-community/cli-plugin-metro/build/tools/loadMetroConfig.js
+++ b/node_modules/@react-native-community/cli-plugin-metro/build/tools/loadMetroConfig.js
@@ -82,6 +82,7 @@ async function loadMetroConfig(ctx, options = {}) {
     (0, _getDefaultMetroConfig.default)(ctx));
     return (0, _metroConfig().mergeConfig)(loadedConfig, overrideConfig);
   }
+  if (!overrideConfig.resolver.resolveRequest) delete overrideConfig.resolver.resolveRequest;
   return (0, _metroConfig().mergeConfig)(await (0, _metroConfig().loadConfig)({
     cwd: ctx.root,
     ...options
