diff --git a/node_modules/@react-native-community/toolbar-android/android/build.gradle b/node_modules/@react-native-community/toolbar-android/android/build.gradle
index e414765..cfd6b9f 100644
--- a/node_modules/@react-native-community/toolbar-android/android/build.gradle
+++ b/node_modules/@react-native-community/toolbar-android/android/build.gradle
@@ -1,15 +1,3 @@
-
-buildscript {
-    repositories {
-        google()
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.2'
-    }
-}
-
 apply plugin: 'com.android.library'
 
 def safeExtGet(prop, fallback) {
