diff --git a/node_modules/html-to-text/lib/formatter.js b/node_modules/html-to-text/lib/formatter.js
index d6cb1e6..68d0337 100644
--- a/node_modules/html-to-text/lib/formatter.js
+++ b/node_modules/html-to-text/lib/formatter.js
@@ -1,4 +1,4 @@
-const he = require('he');
+const {decode} = require('entities');
 
 const { get, numberToLetterSequence, numberToRoman, splitClassesAndIds, trimCharacter } = require('./helper');
 
@@ -146,7 +146,7 @@ function withBrackets (str, brackets) {
 function formatImage (elem, walk, builder, formatOptions) {
   const attribs = elem.attribs || {};
   const alt = (attribs.alt)
-    ? he.decode(attribs.alt, builder.options.decodeOptions)
+    ? decode(attribs.alt, builder.options.decodeOptions)
     : '';
   const src = (!attribs.src)
     ? ''
@@ -176,7 +176,7 @@ function formatAnchor (elem, walk, builder, formatOptions) {
     href = (formatOptions.baseUrl && href[0] === '/')
       ? formatOptions.baseUrl + href
       : href;
-    return he.decode(href, builder.options.decodeOptions);
+    return decode(href, builder.options.decodeOptions);
   }
   const href = getHref();
   if (!href) {
diff --git a/node_modules/html-to-text/lib/html-to-text.js b/node_modules/html-to-text/lib/html-to-text.js
index 9ebda73..330a28c 100644
--- a/node_modules/html-to-text/lib/html-to-text.js
+++ b/node_modules/html-to-text/lib/html-to-text.js
@@ -1,6 +1,6 @@
 const { hp2Builder } = require('@selderee/plugin-htmlparser2');
 const merge = require('deepmerge');
-const he = require('he');
+const {decode} = require("entities");
 const htmlparser = require('htmlparser2');
 const selderee = require('selderee');
 
@@ -377,7 +377,7 @@ function recursiveWalk (walk, dom, builder) {
   for (const elem of dom) {
     switch (elem.type) {
       case 'text': {
-        builder.addInline(he.decode(elem.data, options.decodeOptions));
+        builder.addInline(decode(elem.data, options.decodeOptions));
         break;
       }
       case 'tag': {
