diff --git a/node_modules/jetifier/index.js b/node_modules/jetifier/index.js
index ae3979c..ed609e9 100644
--- a/node_modules/jetifier/index.js
+++ b/node_modules/jetifier/index.js
@@ -6,7 +6,7 @@ const cpus = require('os').cpus().length;
 
 const arg = process.argv.slice(2)[0];
 const mode = arg && ((arg === 'reverse') || (arg === '-r')) ? 'reverse' : 'forward';
-const SEARCH_DIR = 'node_modules';
+const SEARCH_DIR = '../node_modules';
 
 const classesMapping = getClassesMapping();
 const files = readDir(SEARCH_DIR);
