diff --git a/node_modules/react-native-cli-bump-version/react-native.config.js b/node_modules/react-native-cli-bump-version/react-native.config.js
index fe5274e..6197368 100644
--- a/node_modules/react-native-cli-bump-version/react-native.config.js
+++ b/node_modules/react-native-cli-bump-version/react-native.config.js
@@ -10,16 +10,14 @@ module.exports = {
                 console.log("My work here is done.");
                 return;
             }
-
             const appGradlePath = path.join(
                 config.project.android.sourceDir,
                 config.project.android.appName,
                 "build.gradle",
             );
-
             versioner({
                 root: config.root,
-                pbxprojPath: config.project.ios.pbxprojPath,
+                pbxprojPath: `${config.project.ios.sourceDir}/${config.project.ios.xcodeProject.name.replace(".xcworkspace",".xcodeproj")}/project.pbxproj`,
                 buildGradlePath: appGradlePath,
                 type: args.type,
                 semver: args.semver,
