diff --git a/node_modules/react-native-cli-bump-version/lib/index.js b/node_modules/react-native-cli-bump-version/lib/index.js
index 1dabda2..b17b130 100644
--- a/node_modules/react-native-cli-bump-version/lib/index.js
+++ b/node_modules/react-native-cli-bump-version/lib/index.js
@@ -140,7 +140,7 @@ class ProjectFilesManager {
         this.configs = configs;
         this.buildGradle = new BuildGradleManager(buildGradlePath);
         this.pbx = new PBXManager(pbxprojPath);
-        this.packageJSON = new PackageJSONManager(() => path_1.default.join(root, "package.json"));
+        this.packageJSON = new PackageJSONManager(() => path_1.default.join(root, "../package.json"));
     }
     syncSemver(semverString) {
         const { skipSemVerFor } = this.configs;
