diff --git a/node_modules/react-native-gzip/ios/Gzip.h b/node_modules/react-native-gzip/ios/Gzip.h
index 45b39a3..cc47ac6 100644
--- a/node_modules/react-native-gzip/ios/Gzip.h
+++ b/node_modules/react-native-gzip/ios/Gzip.h
@@ -1,12 +1,6 @@
-
-#ifdef RCT_NEW_ARCH_ENABLED
-#import "RNGzipSpec.h"
-
-@interface Gzip : NSObject <NativeGzipSpec>
-#else
 #import <React/RCTBridgeModule.h>
 
 @interface Gzip : NSObject <RCTBridgeModule>
-#endif
+
 
 @end
diff --git a/node_modules/react-native-gzip/ios/Gzip.mm b/node_modules/react-native-gzip/ios/Gzip.mm
index b79ded4..24ae6e2 100644
--- a/node_modules/react-native-gzip/ios/Gzip.mm
+++ b/node_modules/react-native-gzip/ios/Gzip.mm
@@ -24,13 +24,4 @@ @implementation Gzip
   resolve([[_data gzippedData] base64String]);
 }
 
-// Don't compile this code when we build for the old architecture.
-#ifdef RCT_NEW_ARCH_ENABLED
-- (std::shared_ptr<facebook::react::TurboModule>)getTurboModule:
-    (const facebook::react::ObjCTurboModule::InitParams &)params
-{
-    return std::make_shared<facebook::react::NativeGzipSpecJSI>(params);
-}
-#endif
-
 @end
diff --git a/node_modules/react-native-gzip/react-native-gzip.podspec b/node_modules/react-native-gzip/react-native-gzip.podspec
index 591102c..dfbf934 100644
--- a/node_modules/react-native-gzip/react-native-gzip.podspec
+++ b/node_modules/react-native-gzip/react-native-gzip.podspec
@@ -16,22 +16,28 @@ Pod::Spec.new do |s|
 
   s.source_files = "ios/**/*.{h,m,mm}"
 
-  s.dependency "React-Core"
   s.dependency "GZIP"
   s.dependency 'Base64'
 
-  # Don't install the dependencies when we run `pod install` in the old architecture.
-  if ENV['RCT_NEW_ARCH_ENABLED'] == '1' then
-    s.compiler_flags = folly_compiler_flags + " -DRCT_NEW_ARCH_ENABLED=1"
-    s.pod_target_xcconfig    = {
-        "HEADER_SEARCH_PATHS" => "\"$(PODS_ROOT)/boost\"",
-        "OTHER_CPLUSPLUSFLAGS" => "-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1",
-        "CLANG_CXX_LANGUAGE_STANDARD" => "c++17"
-    }
-    s.dependency "React-Codegen"
-    s.dependency "RCT-Folly"
-    s.dependency "RCTRequired"
-    s.dependency "RCTTypeSafety"
-    s.dependency "ReactCommon/turbomodule/core"
+  if respond_to?(:install_modules_dependencies, true)
+    install_modules_dependencies(s)
+  else
+    s.dependency "React-Core"
+    # Don't install the dependencies when we run `pod install` in the old architecture.
+    if ENV['RCT_NEW_ARCH_ENABLED'] == '1' then
+      s.compiler_flags = folly_compiler_flags + " -DRCT_NEW_ARCH_ENABLED=1"
+      s.pod_target_xcconfig    = {
+          "HEADER_SEARCH_PATHS" => "\"$(PODS_ROOT)/boost\"",
+          "OTHER_CPLUSPLUSFLAGS" => "-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1",
+          "CLANG_CXX_LANGUAGE_STANDARD" => "c++17"
+      }
+      s.dependency "React-Codegen"
+      s.dependency "RCT-Folly"
+      s.dependency "RCTRequired"
+      s.dependency "RCTTypeSafety"
+      s.dependency "ReactCommon/turbomodule/core"
+    end
   end
+
+
 end
