diff --git a/node_modules/react-native-actions-sheet/dist/src/index.js b/node_modules/react-native-actions-sheet/dist/src/index.js
index 0e124da..3703153 100644
--- a/node_modules/react-native-actions-sheet/dist/src/index.js
+++ b/node_modules/react-native-actions-sheet/dist/src/index.js
@@ -1088,6 +1088,7 @@ export default forwardRef(function ActionSheet(_a, ref) {
                  * Always true, it causes issue with keyboard handling.
                  */
                 statusBarTranslucent: true,
+                navigationBarTranslucent: true
             }
             : {
                 testID: ((_b = props.testIDs) === null || _b === void 0 ? void 0 : _b.root) || props.testID,
