diff --git a/node_modules/react-native-device-info/android/build.gradle b/node_modules/react-native-device-info/android/build.gradle
index de22598..7d4dab9 100644
--- a/node_modules/react-native-device-info/android/build.gradle
+++ b/node_modules/react-native-device-info/android/build.gradle
@@ -50,18 +50,6 @@ repositories {
 
 dependencies {
   implementation "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
-  implementation "com.android.installreferrer:installreferrer:${safeExtGet('installReferrerVersion', '1.1.2')}"
-  def firebaseBomVersion = safeExtGet("firebaseBomVersion", null)
-  def firebaseIidVersion = safeExtGet('firebaseIidVersion', null)
-  if (firebaseBomVersion) {
-      implementation platform("com.google.firebase:firebase-bom:${firebaseBomVersion}")
-      implementation "com.google.firebase:firebase-iid"
-  } else if(firebaseIidVersion){
-      implementation "com.google.firebase:firebase-iid:${firebaseIidVersion}"
-  }else{
-      def iidVersion = safeExtGet('googlePlayServicesIidVersion', safeExtGet('googlePlayServicesVersion', '17.0.0'))
-      implementation "com.google.android.gms:play-services-iid:$iidVersion"
-  }
 
   testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.0'
   testImplementation "org.mockito:mockito-core:3.6.28"
diff --git a/node_modules/react-native-device-info/android/src/main/java/com/learnium/RNDeviceInfo/RNInstallReferrerClient.java b/node_modules/react-native-device-info/android/src/main/java/com/learnium/RNDeviceInfo/RNInstallReferrerClient.java
index d02152b..a31992a 100644
--- a/node_modules/react-native-device-info/android/src/main/java/com/learnium/RNDeviceInfo/RNInstallReferrerClient.java
+++ b/node_modules/react-native-device-info/android/src/main/java/com/learnium/RNDeviceInfo/RNInstallReferrerClient.java
@@ -19,9 +19,7 @@ public class RNInstallReferrerClient {
       InstallReferrerClientClazz = Class.forName("com.android.installreferrer.api.InstallReferrerClient");
       InstallReferrerStateListenerClazz = Class.forName("com.android.installreferrer.api.InstallReferrerStateListener");
       ReferrerDetailsClazz = Class.forName("com.android.installreferrer.api.ReferrerDetails");
-    } catch (Exception e) {
-      System.err.println("RNInstallReferrerClient exception. 'installreferrer' APIs are unavailable.");
-    }
+    } catch (Exception e) {}
   }
 
   private final SharedPreferences sharedPreferences;
