diff --git a/node_modules/react-native-svg/__tests__/css.test.tsx b/node_modules/react-native-svg/__tests__/css.test.tsx
index 579c357..57acc09 100644
--- a/node_modules/react-native-svg/__tests__/css.test.tsx
+++ b/node_modules/react-native-svg/__tests__/css.test.tsx
@@ -1,6 +1,7 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
-import { SvgCss, parse, inlineStyles } from '../src/ReactNativeSVG';
+import { parse } from '../src/ReactNativeSVG';
+import { SvgCss, inlineStyles } from '../css';
 
 const xml = `<?xml version="1.0" standalone="no"?>
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
diff --git a/node_modules/react-native-svg/css/index.ts b/node_modules/react-native-svg/css/index.ts
new file mode 100644
index 0000000..3b268af
--- /dev/null
+++ b/node_modules/react-native-svg/css/index.ts
@@ -0,0 +1,2 @@
+export * from '../src/css';
+export { LocalSvg, WithLocalSvg, loadLocalRawResource } from '../src/LocalSvg';
\ No newline at end of file
diff --git a/node_modules/react-native-svg/src/LocalSvg.tsx b/node_modules/react-native-svg/src/LocalSvg.tsx
index f13c46f..0457159 100644
--- a/node_modules/react-native-svg/src/LocalSvg.tsx
+++ b/node_modules/react-native-svg/src/LocalSvg.tsx
@@ -7,7 +7,7 @@ import {
 } from 'react-native';
 
 import { fetchText } from './xml';
-import { SvgCss, SvgWithCss } from './css';
+import { SvgCss, SvgWithCss } from '../css';
 import { SvgProps } from './elements/Svg';
 
 const { getRawResource } = NativeModules.RNSVGRenderableManager || {};
diff --git a/node_modules/react-native-svg/src/ReactNativeSVG.ts b/node_modules/react-native-svg/src/ReactNativeSVG.ts
index 6b72423..1daca6c 100644
--- a/node_modules/react-native-svg/src/ReactNativeSVG.ts
+++ b/node_modules/react-native-svg/src/ReactNativeSVG.ts
@@ -25,14 +25,7 @@ import Marker from './elements/Marker';
 import ForeignObject from './elements/ForeignObject';
 
 import { parse, SvgAst, SvgFromUri, SvgFromXml, SvgUri, SvgXml } from './xml';
-import {
-  SvgCss,
-  SvgCssUri,
-  SvgWithCss,
-  SvgWithCssUri,
-  inlineStyles,
-} from './css';
-import { LocalSvg, WithLocalSvg, loadLocalRawResource } from './LocalSvg';
+
 import {
   RNSVGCircle,
   RNSVGClipPath,
@@ -116,14 +109,6 @@ export {
   SvgFromXml,
   SvgUri,
   SvgXml,
-  SvgCss,
-  SvgCssUri,
-  SvgWithCss,
-  SvgWithCssUri,
-  inlineStyles,
-  LocalSvg,
-  WithLocalSvg,
-  loadLocalRawResource,
   Shape,
   RNSVGMarker,
   RNSVGMask,
