diff --git a/node_modules/react-native-sodium/android/build.gradle b/node_modules/react-native-sodium/android/build.gradle
index 4d717a4..900a7cc 100755
--- a/node_modules/react-native-sodium/android/build.gradle
+++ b/node_modules/react-native-sodium/android/build.gradle
@@ -26,4 +26,5 @@ dependencies {
     implementation "com.goterl:lazysodium-android:5.0.2@aar"
     implementation "net.java.dev.jna:jna:5.9.0@aar"
     implementation 'org.lz4:lz4-java:1.8.0'
+    implementation "androidx.documentfile:documentfile:1.1.0-alpha01"
 }
