diff --git a/node_modules/@tiptap/extension-link/dist/index.cjs b/node_modules/@tiptap/extension-link/dist/index.cjs
index c5a8ab7..fa6e342 100644
--- a/node_modules/@tiptap/extension-link/dist/index.cjs
+++ b/node_modules/@tiptap/extension-link/dist/index.cjs
@@ -101,6 +101,8 @@ function clickHandler(options) {
         key: new state.PluginKey('handleClickLink'),
         props: {
             handleClick: (view, pos, event) => {
+                if (event.button !== 1) return;
+
                 var _a, _b, _c;
                 const attrs = core.getAttributes(view.state, options.type.name);
                 const link = (_a = event.target) === null || _a === void 0 ? void 0 : _a.closest('a');
diff --git a/node_modules/@tiptap/extension-link/dist/index.js b/node_modules/@tiptap/extension-link/dist/index.js
index 3dfe757..dd6d3b5 100644
--- a/node_modules/@tiptap/extension-link/dist/index.js
+++ b/node_modules/@tiptap/extension-link/dist/index.js
@@ -97,6 +97,8 @@ function clickHandler(options) {
         key: new PluginKey('handleClickLink'),
         props: {
             handleClick: (view, pos, event) => {
+                if (event.button !== 1) return;
+                
                 var _a, _b, _c;
                 const attrs = getAttributes(view.state, options.type.name);
                 const link = (_a = event.target) === null || _a === void 0 ? void 0 : _a.closest('a');
diff --git a/node_modules/@tiptap/extension-link/dist/index.umd.js b/node_modules/@tiptap/extension-link/dist/index.umd.js
index c75d4ed..d1d7f33 100644
--- a/node_modules/@tiptap/extension-link/dist/index.umd.js
+++ b/node_modules/@tiptap/extension-link/dist/index.umd.js
@@ -99,6 +99,8 @@
           key: new state.PluginKey('handleClickLink'),
           props: {
               handleClick: (view, pos, event) => {
+                  if (event.button !== 1) return;
+
                   var _a, _b, _c;
                   const attrs = core.getAttributes(view.state, options.type.name);
                   const link = (_a = event.target) === null || _a === void 0 ? void 0 : _a.closest('a');
