diff --git a/node_modules/@tiptap/core/dist/tiptap-core.esm.js b/node_modules/@tiptap/core/dist/tiptap-core.esm.js
index 6ef6d0c..fd89f44 100644
--- a/node_modules/@tiptap/core/dist/tiptap-core.esm.js
+++ b/node_modules/@tiptap/core/dist/tiptap-core.esm.js
@@ -3120,8 +3120,7 @@ const Keymap = Extension.create({
             new Plugin({
                 key: new PluginKey('clearDocument'),
                 appendTransaction: (transactions, oldState, newState) => {
-                    const docChanges = transactions.some(transaction => transaction.docChanged)
-                        && !oldState.doc.eq(newState.doc);
+                    const docChanges = transactions.some(transaction => transaction.docChanged);
                     if (!docChanges) {
                         return;
                     }
@@ -3129,10 +3128,14 @@ const Keymap = Extension.create({
                     const allFrom = Selection.atStart(oldState.doc).from;
                     const allEnd = Selection.atEnd(oldState.doc).to;
                     const allWasSelected = from === allFrom && to === allEnd;
-                    const isEmpty = newState.doc.textBetween(0, newState.doc.content.size, ' ', ' ').length === 0;
-                    if (empty || !allWasSelected || !isEmpty) {
+
+                    if (empty || !allWasSelected) {
                         return;
                     }
+
+                    const isEmpty = newState.doc.textBetween(0, newState.doc.content.size, ' ', ' ').length === 0;
+                    if (!isEmpty) return;
+                    
                     const tr = newState.tr;
                     const state = createChainableState({
                         state: newState,
