diff --git a/node_modules/@tiptap/extension-link/dist/index.cjs b/node_modules/@tiptap/extension-link/dist/index.cjs
index 301cdb9..a12e459 100644
--- a/node_modules/@tiptap/extension-link/dist/index.cjs
+++ b/node_modules/@tiptap/extension-link/dist/index.cjs
@@ -110,7 +110,8 @@ function clickHandler(options) {
                 const href = (_b = link === null || link === void 0 ? void 0 : link.href) !== null && _b !== void 0 ? _b : attrs.href;
                 const target = (_c = link === null || link === void 0 ? void 0 : link.target) !== null && _c !== void 0 ? _c : attrs.target;
                 if (link && href) {
-                    window.open(href, target);
+                    event.preventDefault();
+                    if (view.editable) window.open(href, target);
                     return true;
                 }
                 return false;
diff --git a/node_modules/@tiptap/extension-link/dist/index.js b/node_modules/@tiptap/extension-link/dist/index.js
index e3b8602..46265e6 100644
--- a/node_modules/@tiptap/extension-link/dist/index.js
+++ b/node_modules/@tiptap/extension-link/dist/index.js
@@ -106,7 +106,8 @@ function clickHandler(options) {
                 const href = (_b = link === null || link === void 0 ? void 0 : link.href) !== null && _b !== void 0 ? _b : attrs.href;
                 const target = (_c = link === null || link === void 0 ? void 0 : link.target) !== null && _c !== void 0 ? _c : attrs.target;
                 if (link && href) {
-                    window.open(href, target);
+                    event.preventDefault();
+                    if (view.editable) window.open(href, target);
                     return true;
                 }
                 return false;
