diff --git a/node_modules/prosemirror-view/dist/index.cjs b/node_modules/prosemirror-view/dist/index.cjs
index a615cb7..c1a6cbb 100644
--- a/node_modules/prosemirror-view/dist/index.cjs
+++ b/node_modules/prosemirror-view/dist/index.cjs
@@ -1005,8 +1005,8 @@ var ViewDesc = function () {
       if (!(force || brKludge && safari) && isEquivalentPosition(anchorDOM.node, anchorDOM.offset, selRange.anchorNode, selRange.anchorOffset) && isEquivalentPosition(headDOM.node, headDOM.offset, selRange.focusNode, selRange.focusOffset)) return;
       var domSelExtended = false;
       if ((domSel.extend || anchor == head) && !(brKludge && gecko)) {
-        domSel.collapse(anchorDOM.node, anchorDOM.offset);
         try {
+          domSel.collapse(anchorDOM.node, anchorDOM.offset);
           if (anchor != head) domSel.extend(headDOM.node, headDOM.offset);
           domSelExtended = true;
         } catch (_) {}
@@ -3647,7 +3647,7 @@ function handleDrop(view, event, dragging) {
     });
     tr.setSelection(selectionBetween(view, $pos, tr.doc.resolve(end)));
   }
-  view.focus();
+  if (!dragging || !dragging.nodeView) view.focus();
   view.dispatch(tr.setMeta("uiEvent", "drop"));
 }
 handlers.focus = function (view) {
diff --git a/node_modules/prosemirror-view/dist/index.js b/node_modules/prosemirror-view/dist/index.js
index 61118ee..ec4c853 100644
--- a/node_modules/prosemirror-view/dist/index.js
+++ b/node_modules/prosemirror-view/dist/index.js
@@ -1079,8 +1079,8 @@ class ViewDesc {
         // browsers support it yet.
         let domSelExtended = false;
         if ((domSel.extend || anchor == head) && !(brKludge && gecko)) {
-            domSel.collapse(anchorDOM.node, anchorDOM.offset);
             try {
+                domSel.collapse(anchorDOM.node, anchorDOM.offset);
                 if (anchor != head)
                     domSel.extend(headDOM.node, headDOM.offset);
                 domSelExtended = true;
@@ -3885,7 +3885,7 @@ function handleDrop(view, event, dragging) {
         tr.mapping.maps[tr.mapping.maps.length - 1].forEach((_from, _to, _newFrom, newTo) => end = newTo);
         tr.setSelection(selectionBetween(view, $pos, tr.doc.resolve(end)));
     }
-    view.focus();
+    if (!dragging || !dragging.nodeView) view.focus();
     view.dispatch(tr.setMeta("uiEvent", "drop"));
 }
 handlers.focus = view => {
