diff --git a/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js b/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js
index 739a48c..7a7b4af 100644
--- a/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js
+++ b/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js
@@ -117,6 +117,7 @@ class LinuxTargetHelper {
             StartupWMClass: appInfo.productName,
             ...extra,
             ...targetSpecificOptions.desktop,
+            actions: undefined
         };
         const description = this.getDescription(targetSpecificOptions);
         if (!(0, builder_util_1.isEmptyOrSpaces)(description)) {
@@ -160,6 +161,23 @@ class LinuxTargetHelper {
             data += `\n${name}=${desktopMeta[name]}`;
         }
         data += "\n";
+               
+        if (targetSpecificOptions.desktop.actions) {
+            let actionsData = "";
+            const validActions = [];
+            for (const action of targetSpecificOptions.desktop.actions) {
+                if (!action.id || !action.name || !action.args) continue;
+                actionsData += "\n";
+                actionsData += `[Desktop Action ${action.id}]
+Name=${action.name}
+Exec=${desktopMeta.Exec} ${action.args}`;
+                actionsData += "\n";
+
+                validActions.push(action.id);
+            }
+
+            data += `Actions=${validActions.join(";")};\n${actionsData}`
+        }
         return Promise.resolve(data);
     }
 }
