diff --git a/node_modules/electron-trpc/dist/main.mjs b/node_modules/electron-trpc/dist/main.mjs
index 379cf3b..2644e5d 100644
--- a/node_modules/electron-trpc/dist/main.mjs
+++ b/node_modules/electron-trpc/dist/main.mjs
@@ -221,9 +221,16 @@ class G {
     i(this, c).includes(r) || (i(this, c).push(r), I(this, T, W).call(this, r));
   }
   detachWindow(r) {
+    
     y(this, c, i(this, c).filter((n) => n !== r));
-    for (const [n, t] of i(this, u).entries())
-      n.startsWith(`${r.webContents.id}-`) && (t.unsubscribe(), i(this, u).delete(n));
+    for (const [n, t] of i(this, u).entries()) {
+      try {
+        n.startsWith(`${r.webContents.id}-`) && (t.unsubscribe(), i(this, u).delete(n));
+      } catch(e) {
+        console.error(e);
+        // ignore
+      }
+    }
   }
 }
 c = new WeakMap(), u = new WeakMap(), T = new WeakSet(), W = function(r) {
