diff --git a/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_duplex.js b/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_duplex.js
index 57003c3..0a0aa7e 100644
--- a/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_duplex.js
+++ b/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_duplex.js
@@ -44,7 +44,8 @@ module.exports = Duplex;

 /*<replacement>*/
 var util = Object.create(require('core-util-is'));
-util.inherits = require('inherits');
+const inherits = require('inherits');
+util.inherits = "default" in inherits ? inherits.default : inherits;
 /*</replacement>*/

 var Readable = require('./_stream_readable');
diff --git a/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_passthrough.js b/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_passthrough.js
index 612edb4..c7be040 100644
--- a/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_passthrough.js
+++ b/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_passthrough.js
@@ -31,7 +31,8 @@ var Transform = require('./_stream_transform');

 /*<replacement>*/
 var util = Object.create(require('core-util-is'));
-util.inherits = require('inherits');
+const inherits = require('inherits');
+util.inherits = "default" in inherits ? inherits.default : inherits;
 /*</replacement>*/

 util.inherits(PassThrough, Transform);
diff --git a/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_readable.js b/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_readable.js
index 3af95cb..d0bd956 100644
--- a/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_readable.js
+++ b/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_readable.js
@@ -39,7 +39,7 @@ var Duplex;
 Readable.ReadableState = ReadableState;

 /*<replacement>*/
-var EE = require('events').EventEmitter;
+var EE = require('node:events').EventEmitter;

 var EElistenerCount = function (emitter, type) {
   return emitter.listeners(type).length;
@@ -65,7 +65,8 @@ function _isUint8Array(obj) {

 /*<replacement>*/
 var util = Object.create(require('core-util-is'));
-util.inherits = require('inherits');
+const inherits = require('inherits');
+util.inherits = "default" in inherits ? inherits.default : inherits;
 /*</replacement>*/

 /*<replacement>*/
diff --git a/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_transform.js b/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_transform.js
index fcfc105..9782b44 100644
--- a/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_transform.js
+++ b/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_transform.js
@@ -69,7 +69,8 @@ var Duplex = require('./_stream_duplex');

 /*<replacement>*/
 var util = Object.create(require('core-util-is'));
-util.inherits = require('inherits');
+const inherits = require('inherits');
+util.inherits = "default" in inherits ? inherits.default : inherits;
 /*</replacement>*/

 util.inherits(Transform, Duplex);
diff --git a/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_writable.js b/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_writable.js
index e1e897f..df0ca6d 100644
--- a/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_writable.js
+++ b/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_writable.js
@@ -65,7 +65,8 @@ Writable.WritableState = WritableState;

 /*<replacement>*/
 var util = Object.create(require('core-util-is'));
-util.inherits = require('inherits');
+const inherits = require('inherits');
+util.inherits = "default" in inherits ? inherits.default : inherits;
 /*</replacement>*/

 /*<replacement>*/