diff --git a/node_modules/multipipe/index.js b/node_modules/multipipe/index.js
index 4d5e227..9323300 100644
--- a/node_modules/multipipe/index.js
+++ b/node_modules/multipipe/index.js
@@ -4,8 +4,8 @@
  */
 
 var duplexer = require('duplexer2');
-var PassThrough = require('stream').PassThrough;
-var Readable = require('stream').PassThrough;
+var PassThrough = require('node:stream').PassThrough;
+var Readable = require('node:stream').PassThrough;
 var objectAssign = require('object-assign');
 
 /**
